.ci/pipeline/ci.yml

```diff
@@ -161,8 +161,9 @@ jobs:
       conda config --set always_yes yes --set changeps1 no
       conda update -q conda
       conda create -n CB python=$(python.version)
-      conda install -n CB -q cython scipy pytest pandas pyyaml joblib numpydoc jinja2 numpy clang-tools lightgbm scikit-learn mpich
+      conda install -n CB -q cython scipy pytest pandas pyyaml joblib numpydoc jinja2 numpy clang-tools lightgbm scikit-learn
       conda install -n CB -q --override-channels -c intel dal dal-include
+      conda install -n CB -q --override-channels -c conda-forge mpich
```

Comment:
These changes were added to sklearnex recently, you should exclude them in this PR/rebase from current develop branch